src/cryptography/x509.py

```diff
@@ -237,6 +237,57 @@ def __repr__(self):
                 "path_length={0.path_length})>").format(self)
 
 
+class KeyUsage(object):
+    def __init__(self, usages):
```

Comment:
No, it can definitely take 9 params and just make them kwargs. That actually sounds like a better solution than having setters.